### DISCUSSION TITLE
Should we Prune our Nighly Releases?

### DISCUSSION THREAD

**Discussion by zicklag** [fishfolk/jumpy#855] [Q&A] [answered]
@erlend-sh and @orhun I was wondering if either of you had any thoughts about whether or not we should prune our nightly releases.

We now have a bunch releases that are identical to each-other because we haven't merged anything to master in a couple months, and it seems slightly weird to keep the nightly releases around forever.

I know some projects do nightly releases and just keep them, like brave which has 5623 releases right now, so maybe it's not a problem, but we're also downloading an index of all of our releases for our launcher, so I'm not sure if that gets unruly at a certain point.

**Comment by erlend-sh** [comment]
I’m in favor of pruning. Ideally we also wouldn’t be pushing releases unless new changes have been merged, although that will probably be less important again now that the refactor is landing and Main is active again.

**Reply by zicklag** [reply]
> Ideally we also wouldn’t be pushing releases unless new changes have been merged

That's a good point. Initially I was thinking that would be difficult, but after a second thought, I don't think that would be too hard to do. If we got that implemented, and we cleaned up all but the latest of our current nightlies, then that might settle us for a while at least, and then we can later setup a specific pruning schedule.

**Comment by orhun** [comment]
Yeah, fair point. I think we can simply disable them for now. +1 from me.

**Reply by orhun** [reply]
Also, how about removing them from the launcher as well?

**Reply by zicklag** [reply, marked as answer]
If we delete the GitHub releases I think it will remove them from the launcher, because our launcher generates the release list from the GitHub API.

Also, I just realized that I already have a way to trigger a nightly build manually, so we could just do that when we do something interesting and we want to do a pre-release.

So I'll do something to delete all of the nightly releases that we have right now, which should remove them from the launcher, and then I'll remove the CI schedule so they aren't automatic anymore, but we can trigger them when desired.

**Reply by orhun** [reply]
Makes sense 👍🏼
